src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory_XmlConsumption.cs

```diff
@@ -75,7 +75,7 @@ public static Task<TestWorkspace> CreateWorkspaceAsync(string xmlDefinition, boo
             return CreateWorkspaceAsync(workspaceElement, completed, openDocuments, exportProvider, workspaceKind).WaitAndGetResult_CanCallOnBackground(CancellationToken.None);
         }
 
-        public static async Task<TestWorkspace> CreateWorkspaceAsync(
+        public static Task<TestWorkspace> CreateWorkspaceAsync(
```

Comment:
Why is this async now? Just leaving it as such so we don't have to change everything else again? I think that's fair, but it might be worth a comment or two.